### ISSUE TITLE
Add the ability to only trigger a request once

### ISSUE THREAD

**Issue by href** [LeadDyno/intercooler-js#226]
Not sure if this should be at all added to Intercooler, but it was useful to me - maybe I'm not alone:

I have a page where a user can click on an "expand" button to show some details about an item. These details can be toggled and should only be loaded once (these details change infrequently). Intercooler is a great fit for this feature:

```html
<div class="details"
     ic-trigger-from="#detail-button-1"
     ic-get-from="https://example.org/details/1" />
```

When a user clicks on the button, the details are loaded. Now Intercooler does not know about my toggle, so naturally it would load the details snippet every time the button was toggled. That still works, but adds unnecessary requests to my application.

So I changed my html to this:

```html
<div class="details"
     ic-trigger-from="#detail-button-1"
     ic-get-from="https://example.org/details/1"
     ic-once />
```

And added a little script to implement `ic-once`:
```javascript
$(document).find('[ic-once]').on('success.ic', function(_e, element) {
    setTimeout(function() {
        element.data('ic-request-in-flight', true);
    }, 0);
});
```

This disables all but the first request. Because of #225 I had to use a bit of a hack to get this to work - otherwise I could probably get it to run by adding my own ic-on-before-trigger handler to stop the request from happening.

Anyway, maybe this is useful to other people. Also, if anyone knows a better way of accomplishing what I'm trying to do, let me know :)

**Comment by carsongross** [comment]
We support a `changed` modifier on `ic-trigger-on`:

  http://intercoolerjs.org/attributes/ic-trigger-on.html  

What if we supported a `once` modifier as well?  Your HTML would look like this:

```html
<div class="details"
     ic-trigger-from="#detail-button-1"
     ic-trigger-on="click once"
     ic-get-from="https://example.org/details/1"/>
```

**Comment by carsongross** [comment]
Hmm, maybe it should be a prefix, so it reads better:

```html
<div class="details"
     ic-trigger-from="#detail-button-1"
     ic-trigger-on="first click"
     ic-get-from="https://example.org/details/1"/>
```

**Comment by href** [comment]
I would think that 'once' is better than 'first' because I want the request to go through once. That might not be the first request (if the first one fails, the second one should still be made). In other words, a successful request should only be made once.

**Comment by carsongross** [comment]
Cool will implement soon

**Comment by carsongross** [comment]
Available in the 1.2.1 release.

**Comment by href** [comment]
Awesome, thank you! 😃
